src/Api.js

```diff
@@ -0,0 +1,9 @@
+export async function getData(page, pageSize, order){
```

Comment:
```suggestion
export async function getProducts(page, pageSize, order){
```

getData라고 하면 어떤 데이터인지 다른곳에서 import할 때 알 수가 없을 것 같아요!
정확히 어떤 함수인지 명시해주는 것이 좋습니다~

Reply:
변수명이 조금 길어지더라도 명확하게 작성하는 습관들여보겠습니다.